tests/Integrations/Mysqli/MysqliTest.php

```diff
@@ -297,9 +322,31 @@ private function clearDatabase()
     {
         $this->isolateTracer(function () {
             $mysqli = new \mysqli(self::$host, self::$user, self::$password, self::$db);
-            $mysqli->query("DROP TABLE tests");
+            $mysqli->query("DROP TABLE IF EXISTS tests");
             $mysqli->commit();
             $mysqli->close();
         });
     }
+
+    private function assertOneRowInDatabase($table, $wheres)
```

Comment:
Eventually we want to move these utilities to their own class and apply the same check to PDO and Eloquent tests as well. Created a task "Improve database integration tests with tooling to check if queries actually were successfully executed" for that to be scheduled.